<launch>

  <arg name="sim" default="0"/>
  <arg name="config_path" value="$(find multi_mav_manager)/config/crazyflie"/>
  <arg name="launch_path" value="$(find modrotor_sim)/launch/simul/"/>


  <!--
  <node pkg="modrotor_sim" type="cmd_multiplexer.py" name="cmd_multiplexer"/>
-->

  <include file="$(arg launch_path)/crazyflie_sim.launch">
    <arg name="model" value="crazy01"/>
    <arg name="cmd_frequency" value="100"/>
    <arg name="sim" value="$(arg sim)"/>
    <arg name="config_path" value="$(arg config_path)"/>
    <arg name="uri" value="radio://0/30/2M/0xE7E7E7E708"/>
  </include>

    <include file="$(arg launch_path)/crazyflie_sim.launch">
    <arg name="model" value="crazy02"/>
    <arg name="sim" value="$(arg sim)"/>
    <arg name="config_path" value="$(arg config_path)"/>
    <arg name="uri" value="radio://0/100/2M/0xE7E7E7E701"/>
  </include>

<include file="$(arg launch_path)/crazyflie_sim.launch">
    <arg name="model" value="crazy03"/>
    <arg name="sim" value="$(arg sim)"/>
    <arg name="config_path" value="$(arg config_path)"/>
    <arg name="uri" value="radio://0/100/2M/0xE7E7E7E701"/>
  </include>

  <include file="$(arg launch_path)/crazyflie_sim.launch">
    <arg name="model" value="crazy04"/>
    <arg name="sim" value="$(arg sim)"/>
    <arg name="config_path" value="$(arg config_path)"/>
    <arg name="uri" value="radio://0/100/2M/0xE7E7E7E701"/>
  </include>

<include file="$(arg launch_path)/crazyflie_sim.launch">
    <arg name="model" value="crazy05"/>
    <arg name="sim" value="$(arg sim)"/>
    <arg name="config_path" value="$(arg config_path)"/>
    <arg name="uri" value="radio://0/100/2M/0xE7E7E7E701"/>
  </include>

  <include file="$(arg launch_path)/crazyflie_sim.launch">
    <arg name="model" value="crazy06"/>
    <arg name="sim" value="$(arg sim)"/>
    <arg name="config_path" value="$(arg config_path)"/>
    <arg name="uri" value="radio://0/100/2M/0xE7E7E7E701"/>
  </include>


  <node pkg="multi_mav_manager"
      type="multi_mav_services"
      name="multi_mav_services"
      output="screen">
    <rosparam file="$(arg config_path)/multi_mav_manager.yaml"/>
  </node>



</launch>
